.github/labeler.yml

```
@@ -31,5 +31,9 @@
 'Content: Foundations':
   - 'foundations/**/*'
 
+'Content: Markdownlint':
+  - 'markdownlint/**/*'
+  - '*.markdownlint-cli2.jsonc'
```

```suggestion
  - '**/*.markdownlint-cli2.jsonc'
```
Is it necessarily the case that configs will remain top level forever? I don't see a reason why one would not want to label a config in a subdirectory as being to do with MDL, if such a file begins to exist later

---

That was the intention but clearly my typing had other thoughts. Thanks for catching that!